### PR TITLE
[CHK-3510] feat(producer): add producer service

### PR DESCRIPTION
**Pull request by CianoDanilo**
Depends on https://github.com/pagopa/pagopa-payment-wallet-scheduler-service/pull/2
#### List of Changes
Added event dispatching implementation, configuration and testing

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by ciuffagianluca — DISMISSED** [review]
LGTM 🚀

**Review by EmanueleBVtech — DISMISSED** [review]
seems good 🚀

**Review by EmanueleBVtech — APPROVED** [review]

File: src/main/kotlin/it/pagopa/wallet/scheduler/config/OpenTelemetryConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class OpenTelemetryConfiguration {
+
+    @Bean fun agentOpenTelemetrySDKInstance(): OpenTelemetry = GlobalOpenTelemetry.get()
+
+    @Bean
+    fun openTelemetryTracer(openTelemetry: OpenTelemetry): Tracer =
+        openTelemetry.getTracer("pagopa-wallet-service")
```
Comment (pietro-tota) [inline src/main/kotlin/it/pagopa/wallet/scheduler/config/OpenTelemetryConfiguration.kt:17]:
```suggestion
        openTelemetry.getTracer("pagopa-wallet-scheduler-service")
```

File: src/main/kotlin/it/pagopa/wallet/scheduler/service/CdcEventDispatcherService.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    private val walletQueueClient: WalletQueueClient,
+    private val tracingUtils: TracingUtils,
+    private val cdcQueueConfig: CdcQueueConfig,
+    private val retrySendPolicyConfig: RetrySendPolicyConfig
+) {
+
+    private val WALLET_CDC_EVENT_HANDLER_SPAN_NAME = "schedulerWalletEvent"
+    private val logger: Logger = LoggerFactory.getLogger(javaClass)
+
+    fun dispatchEvent(event: LoggingEvent?): Mono<LoggingEvent> =
```
[the remark this answers is not in the document]
Comment (CianoDanilo) [reply src/main/kotlin/it/pagopa/wallet/scheduler/service/CdcEventDispatcherService.kt]: Done in [26696a1](https://github.com/pagopa/pagopa-payment-wallet-scheduler-service/pull/5/commits/26696a175dcc780e3d060eb95d0a9c39d4d4ac46)

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**